### ISSUE TITLE
Coin form reorg

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#2173]
Reorganize the coin forms and show page to match the order from the existing application:

* coin number
* collection
* location
* issue
* Description:
* counter stamp 
* analysis
* public information
* private information
* find: region/state/city
* find date
* feature
* locus 
* number
* find description
* images
* accession
* provenance and references

**Comment by christinach** [comment]
@escowles regarding the coin form are you doing a cross check with one of the pngs in google drive?

**Comment by escowles** [comment]
@christinach Yes, I'm looking at the `issues-form.png`: https://drive.google.com/open?id=19oiEuR0tJSKx37TZKxgf9lq0B1sasEDi

**Comment by christinach** [comment]
I think holding_location is the departmentid. So that means we need departments. which are 14 and they have 2 columns:
```
DepartmentID	DepartmentName
```

**Comment by christinach** [comment]
@escowles what is the image and the append_id attributes about?

**Comment by escowles** [comment]
@christinach `image_resource?` is how the characterization services check whether it should have image derivative processing run or not.  We are not going to port the way the numismatic application handles images — we're going to attach them as filesets like ScannedResources and EphemeraFolders do.
